ncl/ncl_entries/general_applied_math.ipynb

```diff
@@ -0,0 +1,713 @@
+{
```

Comment:
<p>It's a bit repetitive, but I wonder if doing something like <a href="https://ncar.github.io/geocat-applications/ncl/ncl_entries/days_in_month.html#overview" target="_blank">what was done for days_in_month</a> might be good here too e.g. "NCL's <code>abs</code> returns..." just so it's clear this is the NCL function. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>